tests/unit_tests/test_routes.py

```diff
@@ -35,9 +35,16 @@ def test_create_shortlink_ok(self):
         shorturl = response.json.get('shorturl')
         self.assertEqual('http://localhost/' in shorturl, True)
         short_id = shorturl.replace('http://localhost/', '')
-        self.assertIsNotNone(
-            re.search("^[0-9A-Za-z-_]{" + str(SHORT_ID_SIZE) + "}$", short_id),
-            msg=f'Short ID {short_id} doesn\'t match regex'
+        self.assertEqual(
+            len(short_id),
+            int(SHORT_ID_SIZE),
+            msg=
+            f"Length of short_id '{short_id}' does not match configured size of {SHORT_ID_SIZE} characters"  # pylint: disable=line-too-long
```

Comment:
Breaks the line in 2 here and don't disable pylint warning. IMHO the only reason to disable line-too-long pylint would be for regex and urls.

```python
msg=f"Length of short_id '{short_id}' does not match configured size " 
f"of {SHORT_ID_SIZE} characters"
```

Reply:
:+1:
When I saw the pylint pragma here, I could almost hear @ltshb voice mentioning, to better split the line than disabling the warning :wink:

Reply:
Done